spring-components/rdf4j-spring/pom.xml

```diff
@@ -65,6 +65,12 @@
 			<artifactId>commons-pool2</artifactId>
 			<version>2.8.1</version>
 		</dependency>
+		<dependency>
+			<groupId>com.github.tomakehurst</groupId>
+			<artifactId>wiremock-jre8</artifactId>
+			<version>2.34.0</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
This needs a CQ verification.

Reply:
Looks like we are already using an older version of wiremock through dependency management.

@patrickwyler If you could remove the version line for wiremock (line 71) then it should pick up the version that we are currently using in RDF4J.

Reply:
Done @hmottestad.